### DISCUSSION TITLE
[Question] About Binary Phase Envelope Tutorial for SLE calculations

### DISCUSSION THREAD

**Discussion by pedro-ricardo** [DanWBR/dwsim#635] [General]
Hello there,

I'm starting to know the software and was looking for a tutorial on _SLE_ using the _Binary Phase Envelope_.

The way I set it up for now give me the following error: `The given array is too small. It must be at least 1 long. Parameter name: x`

This seems like something simple that I am forgetting or just setting wrong and not a bug at all.

## Current Configurations
This is the Binary Envelope screen:
![image](https://github.com/DanWBR/dwsim/assets/15893711/7c4af19c-a22a-451f-8993-302e2e2ab1a1)

This is the Property Package:
![image](https://github.com/DanWBR/dwsim/assets/15893711/e2543244-8c26-473d-8e4b-271f45797cb3)

## Expected results:
![image](https://github.com/DanWBR/dwsim/assets/15893711/d36dcedd-234f-4feb-996f-6cb67048337d)

**Comment by DanWBR** [comment]
@pedro-ricardo can you attach the flowsheet?
